core/sds-aspect-model-java-generator/src/main/resources/java-static-class-property-lib.vm

```diff
@@ -337,7 +335,7 @@
           $util.toConstant( $property.getName() ) = new StaticConstraintProperty<$propertyType, #getConstraintClassName(), #getCharacteristicClassName()>(
     #end
 #else
-    #if( $util.hasContainerType( $property ) )
+    #if( $util.hasContainerType( $property ) && !$propertyType.startsWith( "Map" ) )
```

Comment:
Why are you excluding `Map` here?

Reply:
This is a fix to align this section with the generation of the `getContainedType` method starting in line 392. Map is (correctly) marked as a "container type", which is required for example in the generation of the `getPropertyType` method, which in this case returns a `Class<Map>`. The reason Map is excluded here is because maps are generated for Properties such as `bamm:description`, which are represented as a `Map<Locale, String>` in the static class, but for which no `StaticContainerProperty`s should be generated but rather `StaticProperty`s as the are no containers like collections.